pkg/cmd/run/view/view.go

```diff
@@ -249,13 +257,20 @@ func runView(opts *ViewOptions) error {
 		return opts.Browser.Browse(url)
 	}
 
+	if err := opts.IO.StartPager(); err == nil {
```

Comment:
Moved `StartPager()` below the `Web` mode check. It's more consistent with other commands with `--web` this way.

Not really related to this PR scope but I edited `StartPager()` around this so I think it makes sense.